libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -274,4 +281,11 @@ export class DtRadialChart implements AfterContentInit, OnDestroy {
       value: series.value,
     };
   }
+
+  /**
+   * Gets a safe CSS
+   */
```

Comment:
```suggestion
  /**
   * Sanitization of the custom property is necessary as, custom property assignments do not work
   * in a viewEngine setup. This can be removed with angular version 10, if ivy is no longer opt out. 
  */
```